stylesheet.css

```diff
@@ -0,0 +1,32 @@
+/* Menu */
```

Comment:
Can we add default class to parent element in initializer and make default css (or less, or sass) for it. It should make using of this lib a bit easier.